tests/success/expected_results.json

```diff
@@ -1,4 +1 @@
-{
-  "version": 1,
-  "status": "pass"
-}
+{
  "status": "pass",
  "tests": [
    {
      "message": "",
      "name": "Test_Success",
      "status": "pass",
      "test_code": "     ∇ r←Test_Success\r[1]    2 ⎕FIX'Success.aplf'\r[2]    2 ⎕FIX'../Assert.aplf'\r[3]    'Must return 1'Assert 1≡Success ⍬\r[4]    r←1\r     ∇"
    }
  ],
  "version": 2
}
```

Comment:
Could also update the `version` of the other test files? The version is something global to the test runner.

Reply:
I was going to work on those after this merged (#4). Would you prefer I get all the basic tests going first? The code in this PR only runs the success test.

Reply:
Ah, it's fine to update it later then

Reply:
Updated now all tests have been implemented